pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -84,6 +86,10 @@ type TriggerSpec struct {
 	// Subscriber is the addressable that receives events from the Broker that pass the Filter. It
 	// is required.
 	Subscriber duckv1.Destination `json:"subscriber"`
+
+	// Delivery contains the delivery spec for this specific trigger.
+	// +optional
+	Delivery *eventingduckv1.DeliverySpec `json:"delivery,omitempty"`
```

Comment:
Can you add this to the CRD schema https://github.com/knative/eventing/blob/master/config/core/resources/broker.yaml?